code/modules/mob/living/carbon/human/species/station/traits_vr/negative.dm

```diff
@@ -96,6 +96,30 @@
 	cost = -1
 	var_changes = list("flash_mod" = 2.0)
 
+/datum/trait/hollow
+	name = "Hollow Bones/Aluminum Alloy"
+	desc = "Your bones and robot limbs are much easier to break."
+	cost = -4
```

Comment:
I'll have to start a discussion with admins about the cost, since that's a very high point return, considering most people on the server don't get into combat, or get injured.

Reply:
This was the suggested point value for it, since without osteodaxon/pink slime, one or two hits from anything will result in cracked bones. This makes it even more brutal than a 50% damage increase, since at least with those you can bandage it. 

Also, this makes a broken bone from falling any distance exceedingly likely. A single slime swat is nearly guaranteed to break something. Heck, most mobs can break any of these in one or two hits as well. 

Basically, so far as combat drawbacks go, this almost guarantees that if you get into combat, you go to medbay right after, if you're not getting floored from agony, or starting to bleed out from jostling the bones around. I consider that a sizable curse to carry; it basically makes melee combat impossible.

Reply:
I'd be interested to see this maybe have a lower point return, BUT reduce the nutrition costs for flying. Because you're lighter.

Reply:
Synth limbs don't break, so I'd be interested in making sure synths can't take this, because it would have no effect. Otherwise there's no reason every synth won't just take this for automatic haste+ affordability. Also interested in maybe just reducing the cost to -3, but still talking about it with others.

Reply:
Wait, I thought that synth limbs did go into that sparking mess where they drop items/fall over from the damages? Does it not check min_broken for that?

Reply:
Reduce to -2.